### PR TITLE
Fix data transmission issues on AdYouLike bid adapter

### PR DESCRIPTION
**Pull request by guiann**
## Type of change

- Bugfix

## Description of change
Avoid data transition issues : Referrer url was encoded twice and some errors occurs with wrong size values.

- [* ] official adapter submission

### CODE REVIEW

**Comment by guiann** [comment]
Hi @GLStephen,
What are the tests realvuAnalyticsAdapter_spec.js    about ?
 
Should not be related to the current change, should it ?

File: modules/adyoulikeBidAdapter.js [hunk 1]
```diff
@@ -72,11 +72,19 @@ export const spec = {
...
+    try {
+      bidRequests = JSON.parse(request.data).Bids;
+    } catch (e) {
+      // json error initial request can't be read
+    }
+
     // For this adapter, serverResponse is a list
     serverResponse.body.forEach(response => {
-      const bid = createBid(response);
+      const bid = createBid(response, bidRequests);
```
[the remark this answers is not in the document]
Comment (guiann) [reply modules/adyoulikeBidAdapter.js]: I've added a check on bidRequests at this point.
see;  bebf9c2

**Comment by guiann** [comment]
Hi @GLStephen  the requested changes have been made.
Can you please have a look?

**Comment by guiann** [comment]
Hi @GLStephen the requested changes have been made.
Can you please have a look?

**Comment by GLStephen** [comment]
LGTM

**Comment by GLStephen** [comment]
@guiann it looks like conflicts have appeared from master. resolve them and I'll recheck.

**Comment by guiann** [comment]
master merged, conflict resolved. thx

**Comment by guiann** [comment]
Can we have the information on when this will be merged please ?

**Pull request merged**